src/Map/src/Bridge/Google/README.md

```diff
@@ -0,0 +1,75 @@
+# Symfony UX Map: Google
+
+[Google Maps](https://developers.google.com/maps/documentation/javascript/overview) integration for Symfony UX Map.
+
+## DSN example
+
+```dotenv
+UX_MAP_DSN=google://GOOGLE_MAPS_API_KEY@default
+
+# With options
+UX_MAP_DSN=google://GOOGLE_MAPS_API_KEY@default?version=weekly
+UX_MAP_DSN=google://GOOGLE_MAPS_API_KEY@default?language=fr&region=FR
+```
+
+Available options:
+
+| Option     | Description                                                                                                                        | Default                                   |
+|------------|------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
+| `id`       | The id of the script tag                                                                                                           | `__googleMapsScriptId`                    |
+| `language` | Force language, see [list of supported languages](https://developers.google.com/maps/faq#languagesupport) specified in the browser | The user's preferred language             |
+| `region`   | Unicode region subtag identifiers compatible with [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1)                           |                                           |
+| `nonce`    | Use a cryptographic nonce attribute                                                                                                |                                           |
+| `retries`  | The number of script load retries                                                                                                  | 3                                         |
+| `url`      | Custom url to load the Google Maps API script                                                                                      | `https://maps.googleapis.com/maps/api/js` |
+| `version`  | The release channels or version numbers                                                                                            | `weekly`                                  |
```

Comment:
I'd put this at the end of the file (advanced instanciation or something)

For now i fear it pushes too much the default/standard install use cases..